### PR TITLE
test-outside-test-package: functions prefixed test_ aren't tests

### LINKED ISSUE #176: False positive for test-outside-test-package when function named test_*
Functions aren't evaluated as tests, so these should not count as such.

### PR DESCRIPTION
**Pull request by anderseknert**
Fixes #176

### CODE REVIEW

File: bundle/regal/ast.rego [hunk 1]
```diff
@@ -20,6 +20,7 @@ package_name := concat(".", [path.value |
 
 tests := [rule |
 	some rule in input.rules
+	not rule.head.args
```
Comment (charlieegan3) [inline bundle/regal/ast.rego:23]: Does this warrant a comment to say that this is saying `if not function`?

**Review by charlieegan3 — APPROVED** [review]

File: bundle/regal/ast.rego (hunk 1 above)
Comment (anderseknert) [reply bundle/regal/ast.rego:23]: Yeah, I guess I've become a little blind to these things by now 😄 Perhaps even better to add a `is_function(rule)` helper to `ast.rego`? I can do so in a follow-up PR.

Comment (srenatus) [reply bundle/regal/ast.rego:23]: 💭 Or even
```rego
is_function(rule) if not rule.head.args
```

Comment (charlieegan3) [reply bundle/regal/ast.rego:23]: Yeah that works better!

**Pull request merged**